lib/src/presentation/view/movie_home_page.dart

```diff
@@ -17,18 +22,32 @@ class MovieHomePage extends StatelessWidget {
       initialPage: args.index,
     );
     final List<MovieEntity> movies = args.movies;
-    return PageView.builder(
-      controller: _pageController,
-      itemCount: movies.length,
-      itemBuilder: (
-        BuildContext context,
-        int index,
-      ) {
-        return Scaffold(
-          appBar: MovieAppBar(movieTitle: movies[index].title),
-          body: MovieBody(movie: movies[index]),
-        );
-      },
-    );
+    return MultiProvider(
+        providers: [
+          Provider(
+              create: (_) => GenreRemoteRepository(
+                  apiService: Provider.of<ApiService>(context))),
+          Provider(
+              create: (ctx) => GenreUseCase(
```

Comment:
As a suggestion try to change `ctx` with `context` and also add the type of the parameters on the definition